testsuite/tests/test/default-success/test.py

```diff
@@ -0,0 +1,23 @@
+"""
+Run a passing test
+"""
+
+import os.path
+
+from drivers.alr import run_alr
+from drivers.asserts import assert_match
+
+run_alr("init", "--lib", "xxx")
+os.chdir("xxx")
+
+with open("./tests/src/tests-example_test.adb", "w") as f:
+   f.write("""procedure Tests.Example_Test is
+begin
+   null;
+end Tests.Example_Test;
+""")
+
+p = run_alr("test")
+assert_match(".*\[ PASS \] tests-example_test.*", p.out)
```

Comment:
At some not distant point I'd like to see timing of tests just like we have in e3-testsuite. All these lines will need modification; saying it now so maybe in the future there's less maintenance to do.

This is the kind of output we get with e3:

`PASS     00m00s help__equivalent_help`

Reply:
Currently this is the only line with this specific regex to match the output. I think it's best to leave it as is and change it when needed?